ichnaea/data/tests/test_station.py

```diff
@@ -101,29 +103,32 @@ def mock_area(self, updated_areas, key, num=num, session2=session2):
             if num[0] == 2:
                 session2.rollback()
 
-        try:
-            CellUpdater._retry_wait = 0.0001
-            session.execute("set session innodb_lock_wait_timeout = 1")
-            with mock.patch.object(CellUpdater, "add_area_update", mock_area):
-                self.queue_and_update(celery, [obs])
-
-            # the inner task logic was called exactly twice
-            assert num[0] == 2
-
-            shard = CellShard.shard_model(obs.cellid)
-            cells = session.query(shard).all()
-            assert len(cells) == 1
-            assert cells[0].samples == 1
-
-            self.check_statcounter(redis, StatKey.cell, 1)
-            self.check_statcounter(redis, StatKey.unique_cell, 1)
-            stats.check(
-                counter=[("data.observation.insert", 1, ["type:cell"])],
-                timer=[("task", 1, ["task:data.update_cell"])],
```

Comment:
These assert there's only one of each of these messages. I need to re-implement that.